products/oracle-database.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Oracle Database
+category: db
+iconSlug: oracle
+permalink: /oracle-database
+versionCommand: show version
+releasePolicyLink: https://www.oracle.com/us/assets/lifetime-support-technology-069183.pdf
+releaseDateColumn: true
+activeSupportColumn: Premier Support
+eolColumn: Extended Support
+
+releases:
+
+-   releaseCycle: "21c"
+    lts: false
+    latest: "21.4.2.0.0"
+    support: 2024-04-30
+    eol: false
+    latestReleaseDate: 2022-12-16
+    releaseDate: 2020-12-01
+    link : "https://docs.oracle.com/en/database/oracle/oracle-database/21/whats-new.html" 
+
+-   releaseCycle: "19c"
+    lts: true
+    latest: "19.17.0.0.221018"
+    support: 2027-04-30
+    eol: false
+    latestReleaseDate: 2023-01-13
+    releaseDate: 2019-02-01
+    link: "https://docs.oracle.com/en/database/oracle/oracle-database/19/rnrdm/database-release-notes.pdf"
+
+-   releaseCycle: "18c"
+    lts: true
+    latest: "19.17.0.0.221018"
+    support: 2027-04-30
+    eol: false
+    latestReleaseDate: 2023-01-13
+    releaseDate: 2019-02-01
+    link: "https://support.oracle.com/knowledge/Oracle%20Cloud/18202104_9.html"
+  
+-   releaseCycle: "12c"
+    lts: false
+    latest: "12.2.0.1.210720"
+    support: 2017-03-01
+    eol: false
+    latestReleaseDate: 2022-04-01
+    releaseDate: 2013-07-01
+    link: "https://docs.oracle.com/en/database/oracle/oracle-database/12.2/ladbn/index.html"
+  
+-   releaseCycle: "11g"
+    lts: false
+    latest: "11.2.0.4"
+    support: 2015-01-01
+    eol: true
+    latestReleaseDate: 2022-09-13
+    releaseDate: 2007-09-01
+    link: "https://support.oracle.com/knowledge/Oracle%20Cloud/2068368_1.html"
```

Comment:
This is the changelog for Oracle Cloud, may be better to use https://docs.oracle.com/cd/E11882_01/relnotes.112/e23558/toc.htm ?